stdpopsim/slim_engine.py

```diff
@@ -503,6 +503,22 @@
         }
     }
 
+    // Setup mutation callbacks.
+    // For each stdpopsim mutation type with an h-s relationship
+    // we have a sequence of assigned SLiM mutation types;
+    // the first is the one that gets produced by mutation,
+    // and the remainder are assigned by a mutation callback.
```

Comment:
```suggestion
    // and the remainder are assigned by a mutation callback.
    // This sequence of mutation types is a subset of the combined 
    // mutation types list, e.g., including neutral mutations, etc.
```

Reply:
I don't actually understand what your clarifying sentence means - which is "this sequence"? What are you trying to clarify?

Reply:
"this sequence [of mutation types]" is "a sequence of assigned SLiM mutation types; the first is the one that gets produced by mutation, and the remainder are assigned by a mutation callback".

The sentence I offered explains that the sequence you wrote about is a subset of a greater sequence, if other types are present.

Reply:
I tried rewriting this to clarify a few times, but it didn't work - I don't think this is the place to explain that basic concept - it's a thing that we explain elsewhere, and this minor comment in the SLiM code isn't the place to go into a detailed description of which things get slim mutation types.